BingMaps/rest-services/autosuggest.md

```diff
@@ -20,11 +20,11 @@ ms.service: "bing-maps"
 > [!NOTE]
 > **Bing Maps Autosuggest API retirement**
 >
-> Bing Maps Autosuggest API is deprecated and has been retired for all free (Basic) account customers. Enterprise account customers can continue to use Bing Maps Autosuggest API until June 30th, 2028. To avoid service disruptions, all enterprise account customers using Bing Maps Autosuggest API will need to be updated to use Azure Maps [Fuzzy Search (with typeahead)](/rest/api/maps/search/get-search-fuzzy) API by June 30th, 2028. An Azure Maps Autosuggest API will be released soon.
+> Bing Maps Autosuggest API is deprecated and has been retired for all free (Basic) account customers. Enterprise account customers can continue to use Bing Maps Autosuggest API until June 30th, 2028. To avoid service disruptions, all enterprise account customers using Bing Maps Autosuggest API will need to be updated to use Azure Maps [Search autosuggest](/rest/api/maps/search/autosuggest) API by June 30th, 2028. An Azure Maps Autosuggest API will be released soon.
```

Comment:
need to be updated to use Azure Maps [Search autosuggest] ==> Should be [Search autocomplete] 
Should this be rest/api/maps/search/**get-geocode-autocomplete**?
Also, please remove last sentence "An Azure Maps Autosuggest API will be released soon."